run

```diff
@@ -3,13 +3,31 @@ set -e
 
 sysroot="$(rustc --print sysroot)"
 
+uname="$(uname -s)"
```

Comment:
Uhhh I would strongly prefer that we not use a variable name that is the exact match of the command name in the shell context.